romanesco/format/graph/adjacencylist_to_networkx.py

```diff
@@ -0,0 +1,5 @@
+from networkx.readwrite.adjlist import read_adjlist
+from StringIO import StringIO
```

Comment:
To avoid regressions when we eventually add python3 support, we should change this to `from six import StringIO`.